dev/requirements.txt

```diff
@@ -3,3 +3,4 @@ jira==1.0.3
 PyGithub==1.26.0
 Unidecode==0.04.19
 sphinx
+pydata_sphinx_theme
```

Comment:
Maybe, is `pydata_sphinx_theme==0.3.1` better to prevent accidental HTML corruption in the future because HTML code and CSS style is impossible to verify automatically.

Reply:
This PR is already pinning it in `dev/create-release/spark-rm/Dockerfile` for release.

Reply:
I thought about this for a while actually especially given the discussion we had before.
`pydata_sphinx_theme` is actually relatively very new and it has [frequent releases](https://github.com/pandas-dev/pydata-sphinx-theme/tags). I would like to not pin the versions for now and see how it goes if that sounds fine to you as well.